### DISCUSSION TITLE
Confluent Schema-Registry

### DISCUSSION THREAD

**Discussion by lzwaan** [Point72/csp#418] [Q&A]
Does CSP support the confluent kafka schema-registry for avro/protobuf?

**Comment by robambalu** [comment]
Sorry I'm not familiar with what that is

**Comment by lzwaan** [comment]
Using a schema registry in combination with kafka will perform much better, you will serialize the messages in avro or protobuf and the datatypes for the k/v pairs are described in the schema-registry, less bandwidth and more performance for heavy pipelines.

**Comment by robambalu** [comment]
First I'm hearing about it so sounds like an interesting feature to have.  Happy to review PRs for this, or you can open a feature request issue to track
